FirebaseSessions/ProtoSupport/Protos/clientanalytics.options

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 Google LLC
```

Comment:
Do we need this file? All the lines seem to be commented out

Reply:
Looks like it'll still generate without it